tutorials/misc/plot_sensor_locations.py

```diff
@@ -0,0 +1,180 @@
+"""
+Working with sensor locations
+=============================
+
+This tutorial describes how to plot sensor locations, and how the physical
+location of sensors is handled in MNE-Python.
+
+.. contents:: Page contents
+   :local:
+   :depth: 2
+
+As usual we'll start by importing the modules we need and loading some
+:ref:`example data <sample-dataset>`:
+"""
+
+import os
+import numpy as np
+import matplotlib.pyplot as plt
+from mpl_toolkits.mplot3d import Axes3D  # noqa
+from mayavi import mlab
+import mne
+
+sample_data_folder = mne.datasets.sample.data_path()
+sample_data_raw_file = os.path.join(sample_data_folder, 'MEG', 'sample',
+                                    'sample_audvis_raw.fif')
+raw = mne.io.read_raw_fif(sample_data_raw_file, preload=True, verbose=False)
+
+###############################################################################
+# About montages and layouts
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# MNE-Python comes pre-loaded with information about the sensor positions of
+# many MEG and EEG systems. This information is stored in *layout files* and
+# *montages*, which are stored in your ``mne-python`` installation directory,
+# in the ``mne/channels/data/layouts`` and ``mne/channels/data/montages``
+# folders (respectively):
+
+data_dir = os.path.join(os.path.dirname(mne.__file__), 'channels', 'data')
+for subfolder in ['layouts', 'montages']:
+    print('\nBUILT-IN {} FILES'.format(subfolder[:-1].upper()))
+    print('======================')
+    print(sorted(os.listdir(os.path.join(data_dir, subfolder))))
+
+###############################################################################
+# .. sidebar:: Computing sensor locations
+#
+#     If you are interested in how standard EEG sensor positions are computed,
+#     the `eeg_positions`_ repository provides code and documentation to this
+#     end.
+#
+# *Layouts* give sensor positions in 2 dimensions (defined by ``x``, ``y``,
+# ``width``, and ``height`` values for each sensor), and are primarily used for
+# illustrative purposes (i.e., making diagrams of approximate sensor
+# positions). In contrast, *montages* give sensor positions in 3D (``x``,
+# ``y``, ``z``, in meters). In addition, montages come in two "flavors" —
+# :class:`~mne.channels.Montage` objects (for idealized sensor montages) and
+# :class:`~mne.channels.DigMontage` (for subject-specific digitizations of
+# sensor positions).
```

Comment:
```suggestion
# sensor positions, based on measurements during the recording session).
```

Although this is what "digitization" implies, adding the "measurement" part to the sentence makes it clear for people not well versed in such terminology

Reply:
Also perhaps it could be clarified that we can compute "layouts" (2D representations) from Montages and DigMontages ... so layouts do not HAVE to be "idealized", they can refer to measured locations (projected from 3D to 2D via some method that should also be referenced with a link perhaps)

Edit: Reading on with this Doc, I realize that my statement above is only true for EEG data. 


Lastly, perhaps you can find a nice way to add that "DigMontages" are not "preloaded" with MNE (unlike some Montages and Layouts, ... because the DigMontages need to be measured and are always different)